common/static/common/js/components/views/feedback.js

```diff
@@ -4,7 +4,8 @@
             "underscore",
             "underscore.string",
             "backbone",
-            "text!common/templates/components/system-feedback.underscore"],
+            "text!common/templates/components/system-feedback.underscore",
+            "jquery.ui.selectable"],
```

Comment:
I'm still a bit nervous about the version incompatibility. What happens in Studio if the full JQuery UI gets loaded first and then the smaller one is loaded which overwrites the parts that overlap. Is there a risk of there being bugs here. It might make the most sense to upgrade JQuery UI itself to the same version, and include selectable in it at the same time. Then on Studio, there would be no need to load both libraries at once into a page. In the LMS it is harder to manage because we don't use JQuery UI on every page.